Automattic/woocommerce-payments
# Update wordpres/components to fix VisuallyHidden styles not loading

Fixes #4015

#### Changes proposed in this Pull Request



Update wordpres/components to fix VisuallyHidden styles not loading.



This PR updates the @wordpress/components dependency to `19.7.0`. It seems the old dependency has a bug where the `VisuallyHidden` component styles are not loaded by `ExternalLink`





#### Testing instructions



1.- Navigate to Payments → Settings.
2.- Scroll down to the "Transactions and deposits" section.
3.- See the text `opens in a new tab`  is now hiding correctly.




-------------------


**Post merge**
